src/internal/symbol/observable.ts

```diff
@@ -1,32 +1,27 @@
 import { root } from '../util/root';
 
-export function getSymbolObservable(context: { Symbol: SymbolConstructor; }): symbol {
-  let $$observable: symbol;
-  let Symbol = context.Symbol;
-
-  if (typeof Symbol === 'function') {
-    if (Symbol.observable) {
-      $$observable = Symbol.observable;
-    } else {
-      $$observable = Symbol('observable');
-      Symbol.observable = $$observable;
-    }
-  } else {
-    $$observable = <any>'@@observable';
+/** Symbol.observable addition */
+/* Note: This will add Symbol.observable globally for all TypeScript users,
+  however, we are no longer polyfilling Symbol.observable */
+declare global {
+  interface SymbolConstructor {
+    observable: symbol;
   }
-
-  return $$observable;
 }
 
-export const observable = getSymbolObservable(root);
+/* NOTE: Warning users that they don't have a Symbol.observable
+   polyfill. It's not something we'd throw on, because it's not
+   really *required*, however, they should be warned to give them a
+   clue as to why they might be seeing errors when trying to convert
+   ObservableLikes to Observables */
+if (!Symbol || !Symbol.observable) {
+  console.warn('RxJS: Symbol.observable does not exist');
+}
 
 /**
- * @deprecated use observable instead
+ * @deprecated use {@link observable}
  */
-export const $$observable = observable;
+export const $$observable = Symbol && Symbol.observable || '@@observable';
```

Comment:
Same as above, prolly easiest to wrap in a function like done in #3394 so only need to check once. Also will allow you to easily make sure the type is of `symbol` instead of `any` as it is in this case.

Reply:
Wrapping it in a function will only make it easier to test. The module body itself should only be called once, no?

Reply:
@benlesh Partially, though it also makes it easier to only need to do the `typeof Symbol` check once.